javascript/modules/client/src/internal/interfaces/plugins.ts

```diff
@@ -105,7 +121,7 @@ type NewTokenParams = {
 
 export type IAddressListPluginInstall = {
   addresses: string[],
-  proposals: IProposalSettings
+  proposals: IPluginSettings
```

Comment:
Since the plugin settings can grow beyond the token scope, would you feel like `settings: IPluginSettings` makes more sense now?

Reply:
Let's also go for it